### DISCUSSION TITLE
What is contour Multi Cluster Best Practices ?

### DISCUSSION THREAD

**Discussion by lou-lan** [projectcontour/contour#4218] [Q&A]
What is contour Multicluster Best Practices ?

**Comment by youngnick** [comment]
Sorry about the delay here, I don't think we really have too much at the moment. We haven't spent much time yet focusing on multicluster, aside from the work that @stevesloka did on [Gimbal](https://github.com/projectcontour/gimbal) a while ago.

Do you have any more specific questions?
